service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/PlacementSubmittedControllerTest.java

```diff
@@ -79,16 +80,60 @@ void shouldSendNotificationWhenAddingNewChildPlacement() throws Exception {
 
         postSubmittedEvent(callbackRequest);
 
+        verify(notificationClient).sendEmail(
+            NEW_PLACEMENT_APPLICATION_NOTIFICATION_TEMPLATE, "admin@family-court.com",
+            expectedTemplateParameters(), String.valueOf(CASE_ID));
+
+        verify(notificationClient, never()).sendEmail(
+            NEW_PLACEMENT_APPLICATION_NOTIFICATION_TEMPLATE, "FamilyPublicLaw+ctsc@gmail.com",
+            expectedTemplateParameters(), String.valueOf(CASE_ID));
```

Comment:
should these not use eq() assertions? Is this actually asserting anything 😅

Reply:
also we spaced down when using eq in other places, up to you what to do here

Reply:
Discussed.